qutebrowser/config/configdata.yml

```diff
@@ -1406,6 +1406,19 @@ tabs.width:
   desc: "Width (in pixels or as percentage of the window) of the tab bar if
     it's vertical."
 
+tabs.min_width:
+  default: -1
+  type:
+    name: Int
+    minval: -1
+    maxval: maxint
+  desc: >-
+    Minimum width (in pixels) of tabs (-1 to use min text contents size for min width).
```

Comment:
"-1 to use min text contents size for min width" seems hard to understand. What about "-1 to use the text's width"?

Though that sounds like it'd still change the behavior from what's done currently (where tabs can get smaller than their text), and I don't think that's actually the case?

Reply:
Yah, this statement does seem a little off. The min size currently is the size of the text if it's an ellipsis. I think it's best to just leave it at "-1 for the default tab sizing behavior", let me know if you think of something better though!